readthedocs/settings/test.py

```diff
@@ -19,6 +19,8 @@ class CommunityTestSettings(CommunityDevSettings):
     ELASTICSEARCH_DSL_AUTOSYNC = False
     ELASTICSEARCH_DSL_AUTO_REFRESH = True
 
+    RTD_BUILD_MEDIA_STORAGE = 'readthedocs.rtd_tests.storage.BuildMediaFileSystemStorageTest'
```

Comment:
Do we really want this to be the default to *all* tests? It seems like it could have a bunch of unintended outcomes.

Reply:
Yeah, this is not correct. I just realize that it breaks other tests. I'm changing this.